File: Selenium/Activities/Activity22.java
import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;
public class Activity22 {

	public static void main(String[] args) {
		   WebDriver driver=new FirefoxDriver();
		
	       driver.get("https://training-support.net/webelements/popups");
			
		   String title=driver.getTitle();
		   System.out.println("Title of the page is: "+title);
		   
		   driver.findElement(By.id("launcher")).click();
		   
		   WebDriverWait wait=new WebDriverWait(driver,Duration.ofSeconds(10));
		   wait.until(ExpectedConditions.elementToBeClickable(By.id("username")));
		   
		   WebElement username=driver.findElement(By.id("username"));
		   WebElement password=driver.findElement(By.id("password"));
		   
		   username.sendKeys("admin");
		   password.sendKeys("password");
		   
		   driver.findElement(By.xpath("//button[text()='Submit']")).click();
		   
		   System.out.println("Message is: "+driver.findElement(By.cssSelector("h2.mt-5")).getText());
		   
		   driver.quit();

	}

}

File: Selenium/Activities/Activity22.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/popups")
    title=driver.title
    print("Title of the page is: ",title)

    wait = WebDriverWait(driver, timeout=10)

    driver.find_element(By.ID,"launcher").click()
    wait.until(EC.element_to_be_clickable((By.ID, "username")))

    username=driver.find_element(By.ID,"username")
    password=driver.find_element(By.ID, "password")

    username.send_keys("admin")
    password.send_keys("password")

    driver.find_element(By.XPATH, "//button[text()='Submit']").click()

    print("Message is: ",driver.find_element(By.CSS_SELECTOR, "h2.mt-5").text)
